cmd/minishift/cmd/root.go

```diff
@@ -45,22 +49,9 @@ import (
 	"github.com/spf13/viper"
 )
 
-var dirs = [...]string{
-	constants.Minipath,
-	constants.MakeMiniPath("certs"),
-	constants.MakeMiniPath("machines"),
-	constants.MakeMiniPath("cache"),
-	constants.MakeMiniPath("cache", "iso"),
-	constants.MakeMiniPath("cache", "oc"),
-	constants.MakeMiniPath("cache", "images"),
-	constants.MakeMiniPath("config"),
-	constants.MakeMiniPath("addons"),
-	constants.MakeMiniPath("logs"),
-	constants.MakeMiniPath("tmp"),
-}
-
 const (
 	showLibmachineLogs    = "show-libmachine-logs"
+	profile               = "profile"
```

Comment:
No need for a constant, as the only place this is used is https://github.com/minishift/minishift/pull/1380/files?diff=unified#diff-caeade0d74523273373d2689861f618eR170

Reply:
```diff
diff --git a/cmd/minishift/cmd/root.go b/cmd/minishift/cmd/root.go
index b08137ad..0dca0237 100644
--- a/cmd/minishift/cmd/root.go
+++ b/cmd/minishift/cmd/root.go
@@ -51,7 +51,6 @@ import (
 
 const (
        showLibmachineLogs    = "show-libmachine-logs"
-       profile               = "profile"
        enableExperimentalEnv = "MINISHIFT_ENABLE_EXPERIMENTAL"
 )
 
@@ -167,7 +166,7 @@ func processEnvVariables() {
 func init() {
        processEnvVariables()
        RootCmd.PersistentFlags().Bool(showLibmachineLogs, false, "Show logs from libmachine.")
-       RootCmd.PersistentFlags().String(profile, constants.DefaultProfileName, "Profile name")
+       RootCmd.PersistentFlags().String("profile", constants.DefaultProfileName, "Profile name")
        RootCmd.AddCommand(configCmd.ConfigCmd)
        RootCmd.AddCommand(cmdOpenshift.OpenShiftCmd)
        RootCmd.AddCommand(hostfolderCmd.HostfolderCmd)
@@ -310,7 +309,7 @@ func setDefaultActiveProfile() {
        if activeProfile == "" {
                err := profileActions.SetActiveProfile(constants.DefaultProfileName)
                if err != nil {
-                       atexit.ExitWithMessage(1, fmt.Sprintf("Error while setting default profile '%s': %s", profile, err.Error()))
+                       atexit.ExitWithMessage(1, fmt.Sprintf("Error while setting default profile '%s': %s", "profile", err.Error()))
                }
 
                // Only set oc context to default profile when user is looking for default profile
```